src/test/java/org/opensearch/knn/indices/ModelGraveyardTests.java

```diff
@@ -57,6 +60,24 @@ public void testStreams() throws IOException {
         assertTrue(testModelGraveyardCopy.contains(testModelId));
     }
 
+    public void testXContentBuilder() throws IOException {
+        Set<String> modelIds = new HashSet<>();
+        String testModelId = "test-model-id";
```

Comment:
nit: call this model id 1 and the next one model id 2

Reply:
updated